docs/mkdocs/docs/NetworkObject/RemoteProcedureCalls/rpc-validation-by-server.md

```diff
@@ -1,17 +1,19 @@
 # RPC Validation by Server
 
-We live in a world where we just can't trust our clients to send us messages that they should be in the way that we wish them to. Clients may want to cheat the system and try to exploit the network for fun or just so that they can win. Because of this we have created a simple method for you to override in order to check any game critical RPCs that are received from the client and verify them. Before continuing with this part of the documentation, please review the Extending Generated Classes documentation to learn more about **partial** classes.
+We live in a world where we just can't trust our clients to send us messages that they should be in the way that we wish them to. Clients may want to cheat the system and try to exploit the network for fun or just so that they can win. Because of this we have created a simple method for you to override in order to check any game critical RPCs that are received from the client and verify them. Before continuing with this part of the documentation, please review the [Extending Generated Classes](/NetworkContractWizard/extending-generated-classes.md) documentation to learn more about **partial** classes.
 
-When we use the Network Contract Wizard (NCW), it will create 2 generated classes; one for our MonoBehaviour and one for our NetworkObject.
+When we use the Network Contract Wizard (NCW), it will create 2 generated classes; one for our `MonoBehaviour` and one for our `NetworkObject`.
 
-The one that we want to focus on in this example is the generated NetworkObject. So let's say that we opened up the Network Contract Wizard (NCW) and we created a contract named **Ball** ; this will generate a NetworkObject class named **BallNetworkObject**. Now in another folder (not in the Generated folder) you will create a new C# script called **BallNetworkObject**. In here you will create your [partial class](https://msdn.microsoft.com/en-us/library/wa80x488.aspx) for the **BallNetworkObj ect** and you will override the **ServerAllowRpc** method like so:
+The one that we want to focus on in this example is the generated `NetworkBehavior`. So let's say that we opened up the Network Contract Wizard (NCW) and we created a contract named **Ball** ; this will generate a `NetworkObject` class named **BallNetworkObject**. 
```

Comment:
Renamed `NetworkObject` here to `NetworkBehavior` here, and in the code example below.